src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java

```diff
@@ -926,6 +892,16 @@ private void succeedingTestForElementInfos(UUID networkUuid, String variantId, E
         JSONAssert.assertEquals(expectedJson, mvcResult.getResponse().getContentAsString(), JSONCompareMode.NON_EXTENSIBLE);
     }
 
+    @SneakyThrows
```

Comment:
todo later : 
now that we've merged all endpoints, I think we could reduce by a lot this test class complexity by merging tests and calling them for all elementType